cmd/main.go

```diff
@@ -169,6 +170,8 @@ func main() {
 		Client:        mgr.GetClient(),
 		Scheme:        mgr.GetScheme(),
 		OADPNamespace: oadpNamespace,
+		// TODO user input
+		SyncPeriod: 5 * time.Minute,
```

Comment:
yeah correct, this would be a user input but we would need a default value.
Have you given any thoughts on how to accept this from the user ? I can think of a new possibilities:
- From `DPA.spec.nonAdmin` struct.
- Create a struct under `DPA.spec.nonAdmin.Args` and accept the value there.
- Introduce a configmap for nacusers/admin users for specifying such things.
- All of the above are extensible enough for similar user inputs in the future NAC world if needed.

Reply:
The minimum should be no less than velero's sync controller IMO.

Reply:
it will be DPA field under nonAdmin 

I would not validate if the value is greater or equal to velero's value, do not know if is simple to be done

Reply:
Velero value isn't customizable AFAICT so if you want to, it can be a hardcoded copy.

Reply:
@kaovilai https://github.com/vmware-tanzu/velero/blob/6ac38cde8556f0fb338f1b00cd64746790b6c73c/pkg/cmd/server/config/config.go#L233
Is this a different sync ?

Reply:
checking.

Reply:
looks like that will eventually set [here](https://github.com/vmware-tanzu/velero/blob/6ac38cde8556f0fb338f1b00cd64746790b6c73c/pkg/controller/backup_sync_controller.go#L477)

and there are code to sync backup to cluster and [code to delete orphaned backups](https://github.com/vmware-tanzu/velero/blob/main/pkg/controller/backup_sync_controller.go#L307) as part of this velero upstream sync. So I think this is the same case.

Velero default is [1 minute](https://github.com/vmware-tanzu/velero/blob/6ac38cde8556f0fb338f1b00cd64746790b6c73c/pkg/cmd/server/config/config.go#L24) So our 5 minutes works. Could be as low as 2 min imo.